module/script.js

```diff
@@ -56,7 +56,7 @@ class Book {
   }
 }
 
```

Comment:
- [ ] it would be better to refactor the addbook and removeBook methods to the Es6 syntax by converting the class into functions in order to complete the requirements